### DISCUSSION TITLE
T-SNE visual analysis for detection tasks

### DISCUSSION THREAD

**Discussion by hjfdsssdg** [open-mmlab/mmselfsup#596] [Q&A]
Hello, is there a tutorial for visual analysis of T-SNE supporting detection models (e.g. Faster RCNN)? Thanks

**Comment by fangyixiao18** [comment]
Currently our script is for backbone(r50) output, did you mean that you need t-sne for output from other part of detection models, like neck or head?

**Reply by hjfdsssdg** [reply]
@fangyixiao18 Thank you for your answer, yes I would like to use it on the classification head, such as the fully connected classification layer at the end of Faster-RCNN

**Reply by fangyixiao18** [reply]
Currently, we don't have such script, maybe you could write it based on our codes by yourself

**Reply by hjfdsssdg** [reply]
Ok,I will try it, thanks
